lib/absinthe/phase/document/arguments/parse.ex

```diff
@@ -50,8 +50,8 @@ defmodule Absinthe.Phase.Document.Arguments.Parse do
     end
   end
 
-  defp build_value(_normalized, %Type.Scalar{}, _context) do
-    {:error, :bad_parse}
+  defp build_value(%{value: value} = _normalized, nil = _schema_node, _context) do
+    {:ok, value}
```

Comment:
This populates the `data:` field on the node so it gets forwarded to the resolver later